prometheus/tests/test.sh.orig

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+hab pkg install --binlink core/bats
+
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static ps
+hab pkg binlink core/busybox-static netstat
+<<<<<<< HEAD
```

Comment:
more conflicts here but also I'm not sure if you intend to check in `test.sh.orig`